bigquery_etl/view/__init__.py

```diff
@@ -194,10 +194,7 @@ def publish(self, target_project=None, dry_run=False):
 
         # avoid checking references since Jenkins might throw an exception:
         # https://github.com/mozilla/bigquery-etl/issues/2246
-        if (
-            any(str(self.path).endswith(p) for p in SKIP_VALIDATION)
```

Comment:
this condition is already filtered at line 191 above

Reply:
Line 191 is for `SKIP_PUBLISHING`, but there are some views, like `telemetry.client_probe_counts_v1`, that we want to publish although their validation would fail (view name != file path).

Running `./bqetl view publish` with this would fail with Error publishing sql/moz-fx-data-shared-prod/telemetry/client_probe_counts_v1/view.sql. Invalid view definition.

Reply:
ah, I get it now. I totally thought this logic was inverted from what it needed. thanks for the clarification!